etc/Dockerfile.mod.pi

```diff
@@ -0,0 +1,52 @@
+ARG BASE_TAG
+ARG BASE_NAME
+ARG HUB_USER
```

Comment:
if these 3 vars are only used once, can they be a single base image arg?

Reply:
Thanks! Simplified this into single arg `BASE_IMG`.